### PR TITLE
Update to the newest messaging-docker-images version

### PR DESCRIPTION
**Pull request by stevenosborne-wf**
### Story:
New version of messaging-docker-images just dropped (based on AL2) - upgrade to it.

### Acceptance Criteria:
- [ ] At least one InfRe Squad 2 member has reviewed and +1'd
- [ ] Code has been tested and results documented
- [ ] Unit tests have been updated
- [ ] Updates to documentation if necessary
- [ ] Verify and document changes to any other Messaging components
- [ ] Pull request made against the 'develop' branch, not master

### Design Notes:
None.

### How To Test:
Tests pass.

### My Test Results:
Tests pass!

#### Reviewers:
@Workiva/product2

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: scripts/skynet/cross/setup_python.sh [hunk 1]
```diff
@@ -11,5 +11,5 @@ else
 fi
 
 pip install -e ".[tornado]"
-python3 /usr/bin/pip3 install Cython==0.27.2
+python3 /usr/bin/pip3 install Cython==0.27.3
```
Comment (brianshannan-wf) [inline scripts/skynet/cross/setup_python.sh]: This was pinned for a reason (https://github.com/Workiva/frugal/pull/936/files#r148286999) is 0.27.3 cool with that?

Comment (stevenosborne-wf) [reply scripts/skynet/cross/setup_python.sh]: Only change from `0.27.2` to `0.27.3` was a fix for python 3.7 - https://github.com/cython/cython/releases/tag/0.27.3

Comment (brianshannan-wf) [reply scripts/skynet/cross/setup_python.sh]: awesome

**Pull request merged**